services/s3/utils.go

```diff
@@ -69,46 +68,58 @@ func (s *Storage) String() string {
 
 // New will create both Servicer and Storager.
 func New(pairs ...typ.Pair) (typ.Servicer, typ.Storager, error) {
-	return newServicerAndStorager(pairs...)
+	f := Factory{}
```

Comment:
Do we need to call `RegisterFactory` at a right place? As I got `service not registered` when using `NewStoragerFromString`.

Reply:
Yep, the `init()` is missing, I will add it back.

Reply:
Fixed in https://github.com/beyondstorage/go-storage/pull/1009